CRM/Admin/Form/Options.php

```diff
@@ -48,6 +50,13 @@ class CRM_Admin_Form_Options extends CRM_Admin_Form {
    */
   public $submitOnce = TRUE;
 
+  /**
+   * The Option Group ID.
+   *
+   * @var int
+   */
+  protected $_gid;
```

Comment:
Maybe mark this as 'internal' if you don't want to go full private

Reply:
added @internal